_static/scripts/part2/memoryobject/simple_memobj.hh

```diff
@@ -42,6 +42,7 @@
  * This memobj is fully blocking (not non-blocking). Only a single request can
  * be outstanding at a time.
  */
+
```

Comment:
Any need for this change?

Reply:
I guess I add this line by mistake. Sorry about that.